src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -286,23 +286,32 @@ public static string RedirectUriMismatch(string expectedUri, string actualUri)
             " See https://aka.ms/msal-net-invalid-client for details. ";
         public const string SSHCertUsedAsHttpHeader = "MSAL was configured to request SSH certificates from AAD, and these cannot be used as an HTTP authentication header. Developers are responsible for transporting the SSH certificates to the target machines.";
 
+        public static string ExperimentalFeature(string methodName)
+        {
+            return string.Format(
+                CultureInfo.InvariantCulture,
+                "The API {0} is marked as experimental. It may change without incrementing the major version of the library. Call .WithExperimentalFeatues() when creating the public / confidential client to bypass this.",
```

Comment:
sp.....WithExperimentalFeatures()

Reply:
should probably have a disclaimer saying that this one might not do anything and should not be used in production?

Reply:
@henrik-me - some of our "experimental" stuff, like opening the system browser using Edge, is already used in production. I'll add the disclaimer and go through our existing stuff to see what we can remove.